src/app/tests/suites/certification/Test_TC_MOD_1_1.yaml

```diff
@@ -77,27 +99,59 @@ tests:
               contains: [4]
 
     - label:
-          "TH reads the optional attribute(OnMode) in AttributeList from the DUT"
-      PICS: MOD.S.A0005
+          "Read the Feature dependent(MOD.S.F00 - DEPONOFF) and optional attribute(OnMode) in AttributeList from the DUT"
+      PICS: MOD.S.A0005 && MOD.S.F00
       command: "readAttribute"
       attribute: "AttributeList"
       response:
           constraints:
               type: list
               contains: [5]
 
-    - label: "Read the global attribute: AcceptedCommandList"
+    - label: "Read the global attribute AcceptedCommandList. Check if it contains the Feature dependent (MOD.S.F02 - DEPCTM) command with id 0x0 (ChangeToMode)"
+      PICS: MOD.S.Afff9 && !MOD.S.F02
       command: "readAttribute"
       attribute: "AcceptedCommandList"
       response:
           constraints:
               type: list
               contains: [0]
 
+
+    - label: "Read the global attribute AcceptedCommandList. Check if it contains the Feature dependent (MOD.S.F01 - EXT) command with id 0x1 (ChangeToModeWithStatus)"
+      PICS: MOD.S.Afff9 && MOD.S.F01
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [1]
+
+    - label: "Read the global attribute AcceptedCommandList. Check if it excludes the Feature dependent (MOD.S.F02 - DEPCTM) command with id 0x0 (ChangeToMode)"
+      PICS: MOD.S.Afff9 && MOD.S.F02
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              excludes: [0]
+
     - label: "Read the global attribute: GeneratedCommandList"
+      PICS: MOD.S.Afff8 && !MOD.S.F01
+      command: "readAttribute"
+      attribute: "GeneratedCommandList"
+      response:
+          value: []
+          constraints:
+              type: list
+              excludes: [2]
+
+    - label: "Read the global attribute: GeneratedCommandList. Check if it contains the Feature dependent (MOD.S.F01 - EXT) command with id 0x2 (ChangeToModeResponse)"
+      PICS: MOD.S.Afff8 && MOD.S.F01
       command: "readAttribute"
       attribute: "GeneratedCommandList"
       response:
           value: []
```

Comment:
I don't see how `value: []` an `contains: [2]` go together.  Is this relying on some sort of test harness bug where `value` is ignored if `constraints` is present or something?  I don't understand how this would pass otherwise.

Reply:
This is a bug and I'll fix it.